cylc/flow/scripts/cylc_ping.py

```diff
@@ -48,7 +48,7 @@ def main(parser, options, suite, task_id=None):
     # cylc ping SUITE
     pclient('ping_suite')  # (no need to check the result)
     if cylc.flow.flags.verbose:
-        host, port = SuiteRuntimeClient.get_location(suite)
+        host, port = pclient.host, pclient.port
```

Comment:
I've stored these `host` and `port` in the `SuiteRuntimeClient`. We create an instance of this class as `pclient` (above in this file's diff :arrow_up:) which will call `get_location`. So I tried to avoid calling that function again as that doesn't appear to be cached.

Happy to revert and use `get_location` if others prefer`. This is to fix the runtime error when running `cylc ping --verbose $SUITE`…